clang/test/CIR/CodeGen/statement-exprs.c

```diff
@@ -0,0 +1,166 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir
```

Comment:
Can you add a test that requires a cleanup related to the scope of the statement expression?

In the incubator, I see `stmt-expr.c`, `stmt-expr.cpp`, and `stmtexpr-init.c`. I'm not sure if we have enough support for everything included in these tests, but they have some interesting cases, including a commented out test for a statement containing a label that you've added support for.

Reply:
Oh! I would have expected that case to be handled now. We don't even issue a diagnostic saying we missed it. That's not good.